app/src/main/java/org/fossasia/openevent/api/APIClient.java

```diff
@@ -37,6 +48,15 @@ public APIClient() {
                 .setConverter(new GsonConverter(gson))
                 .setEndpoint(Urls.BASE_GET_URL_ALT)
                 .setLogLevel(RestAdapter.LogLevel.FULL)
+                .setErrorHandler(new ErrorHandler() {
+                    @Override
+                    public Throwable handleError(RetrofitError cause) {
+                        ErrorDialogEvent errEvent = new ErrorDialogEvent(null, null);
+                        errEvent.ErrorHandlerFunction(cause, errEvent);
```

Comment:
That's because if you look at ErrorHandlerFunction, I'm changing errorType and errorDesc based on the type of error (i.e cause). That's why I first call ErrorDialogEvent and pass null null, then change the member's parameters using the second line.